source/common/config/delta_subscription_impl.h

```diff
@@ -179,19 +162,26 @@ class DeltaSubscriptionImpl : public Subscription,
     }
 
     grpc_stream_.establishNewStream();
-    subscribe(resources);
-    // The attempt stat here is maintained for the purposes of having consistency between ADS and
```

Comment:
It looks like this is referring to the same thing as the TODOs I added. Just how meaningless is update_attempt for ADS? Can we just do nothing with it and let it sit at 0?